pynocular/database_model.py

```diff
@@ -748,3 +812,36 @@ def to_dict(
                 _dict[prop_name] = prop_value
 
         return _dict
+
+
+# Create public class to use to create type checkable DatabaseModels
+class DatabaseModel(_DatabaseModel, BaseModel):
+    pass
+
+
+#-------------- Prototype code to mess around with ----------------
+
+class MyModel(
+    DatabaseModel, table_name="my_table", database_info=DBInfo("type", (("a", "b"),))
+):
+    field: str = ""
+    other: int = 0
+
+class NestedModel(DatabaseModel, table_name = "table2", database_info=DBInfo("type")):
+    if TYPE_CHECKING:
```

Comment:
This is a way to get the nested models to have type checking/IDE support, but it's not needed